### ISSUE TITLE
Common attribute for entry state

### ISSUE THREAD

**Issue by MikaelSmith** [puppetlabs/wash#604]
It's common for VMs, containers, and services to have a running/not-running state. Some operations - particularly exec, and thus reading the filesystem - don't work when a container or VM is not running.

We should add a common attribute for state. It's useful to know whether it's running - and thus can respond to exec and signals - or some other state. I would model this as a boolean running flag, and a free-form string that describes the state.

`ls` should be updated to display the state, and use color or other symbols to denote a running instance.

**Comment by ekinanp** [comment]
Hm, how do you intend to use the separate flag for running?

I think making `state` a string would be useful. People could specify the entry's possible states in the schema, and the color code for each state.

**Comment by MikaelSmith** [comment]
I mentioned a uniform way of showing that an instance is running. Could also early fail Exec based on that.

**Comment by ekinanp** [comment]
Yeah, was mostly asking to confirm the latter. I think we should take a similar approach to `signal` and specify a set of common states (and color codes associated with that state).
